build/pkgs/openblas/spkg-configure.m4

```diff
@@ -83,7 +83,7 @@ SAGE_SPKG_CONFIGURE([openblas], [
                                              if (  10000 * version[0]
                                                    + 100 * version[1]
                                                          + version[2]
-                                                 >=10000 * ]]SAGE_OPENBLAS_LT_VERSION_MAJOR[[
```

Comment:
This part of the script is only used on the Cygwin platform; see lines 54-55.
The change needs to be made in the first branch, before line 51

Reply:
This part applies to non-Cygwin platforms. On Cygwin lines 56-57 apply instead.

Reply:
You're right, sorry for the noise.

Nevertheless the path with pkgconfig should be modified too (pkgconfig is the preferred option, the other just a fallback). Just use 2 calls of `PKG_CHECK_MODULES` to handle 2 ranges